lib/workers/pr/body/updates-table.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { BranchConfig, BranchUpgradeConfig } from '../../types';
+import { getPrUpdatesTable } from './updates-table';
+
+describe('workers/pr/body/updates-table', () => {
+  it('checks results for getPrUpdatesTable', () => {
+    const configObj: BranchConfig = {
+      branchName: 'some-branch',
+      upgrades: [],
+      prBodyColumns: ['Package', 'Type', 'Update', 'Change', 'Pending'],
+    };
```

Comment:
move after upgrades, so you can use a single assignment
```suggestion
    const configObj: BranchConfig = {
      branchName: 'some-branch',
      upgrades: [upgrade0, upgrade1, upgrade2],
      prBodyColumns: ['Package', 'Type', 'Update', 'Change', 'Pending'],
    };
```

Reply:
d